packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -1218,13 +1218,17 @@ module.exports = class ImportEntries {
           break;
         }
         case 'json': {
+          const structuredPTag = '{"type":"p","attrs":{},"children":[{"text":""}]}';
```

Comment:
@abhinav-from-contentstack , can we make this same changes in the ts module as well. In ts we have this json rte handler in utilities

Reply:
https://github.com/contentstack/cli/blob/63413c768ae87a37a8b2eab1bdc9e3a24b8550a1/packages/contentstack-import/src/utils/entries-helper.ts#L376